app/src/main/java/com/swent/assos/ui/screens/manageAsso/CreateEvent.kt

```diff
@@ -203,10 +218,17 @@ fun CreateEvent(assoId: String, navigationActions: NavigationActions) {
                     onValueChange = { viewModel.setDescription(it) },
                     label = { Text("Description") })
                 Spacer(modifier = Modifier.height(16.dp))
-                OutlinedTextField(
-                    value = event.image,
-                    onValueChange = { viewModel.setImage(it) },
-                    label = { Text("Event Image") })
+                Image(
+                    painter = rememberAsyncImagePainter(event.image),
+                    contentDescription = "image",
+                    modifier =
+                        Modifier.size(100.dp)
+                            .background(MaterialTheme.colorScheme.surface)
+                            .clickable {
+                              val pickImageIntent = Intent(Intent.ACTION_PICK)
+                              pickImageIntent.type = "image/*"
+                              launcher.launch(pickImageIntent)
+                            })
```

Comment:
There can be several images so we should display them in the alertDialog `AlertDialogFields`

Reply:
It upload only one image. This PR is just a model, we will implement everywhere in other PR.